### PR TITLE
Autenti E-Signature Standard Workflow Connector initial

### PR DESCRIPTION
**Pull request by ferenchub**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by ferenchub** [comment]
@microsoft-github-policy-service agree company="Autenti"

File: certified-connectors/Autenti E-Signature Standard Workflow Connector/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Autenti E-Signature Standard Workflow Connector for Power Automate
+
```
Comment (vmanoharas) [inline certified-connectors/Autenti E-Signature Standard Workflow Connector/README.md:2]: please format the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md). All of the required headings should be present.

File: certified-connectors/Autenti E-Signature Standard Workflow Connector/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,1083 @@
+{
```
Comment (vmanoharas) [inline certified-connectors/Autenti E-Signature Standard Workflow Connector/apiDefinition.swagger.json:1]: Kindly fix below swagger validation errors. 

2024-04-23T16:17:53.2521423Z ##[error]The 'title' value goes over the character limit '30' ValuePath: /info/title
2024-04-23T16:17:53.2522800Z ##[error]The value 'title' contains at least one of the restricted words: 'api, connector'. ValuePath: /info/title
2024-04-23T16:17:53.2523954Z ##[error]The value 'description' contains at least one of the restricted words: 'power apps, power automate, logic apps'. ValuePath: /info/description
2024-04-23T16:17:53.2525137Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1parties/post/parameters/0
2024-04-23T16:17:53.2526231Z ##[error]The 'description' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1parties/post/parameters/0
2024-04-23T16:17:53.2527300Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1files/get/parameters/0
2024-04-23T16:17:53.2528381Z ##[error]The 'description' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1files/get/parameters/0
2024-04-23T16:17:53.2529438Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1files/post/parameters/0
2024-04-23T16:17:53.2530505Z ##[error]The 'description' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1files/post/parameters/0
2024-04-23T16:17:53.2531566Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1files/post/parameters/1
2024-04-23T16:17:53.2532631Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1files/post/parameters/2
2024-04-23T16:17:53.2533699Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1files/post/parameters/3
2024-04-23T16:17:53.2534757Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1actions/post/parameters/0
2024-04-23T16:17:53.2535816Z ##[error]The 'description' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1actions/post/parameters/0
2024-04-23T16:17:53.2536899Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1files~1{fileId}~1content/get/parameters/0
2024-04-23T16:17:53.2537992Z ##[error]The 'description' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1files~1{fileId}~1content/get/parameters/0
2024-04-23T16:17:53.2539220Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1files~1{fileId}~1content/get/parameters/1
2024-04-23T16:17:53.2540684Z ##[error]The 'description' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1files~1{fileId}~1content/get/parameters/1
2024-04-23T16:17:53.2541707Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1applications~1callbacks/post/parameters/0
2024-04-23T16:17:53.2542697Z ##[error]The 'description' property is required. ValuePath: /paths/~1applications~1callbacks/post/parameters/0
2024-04-23T16:17:53.2543757Z ##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1document-processes/get/responses/default
2024-04-23T16:17:53.2544900Z ##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1document-processes/post/responses/default
2024-04-23T16:17:53.2546077Z ##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1document-processes~1{documentProcessId}~1parties/post/responses/default
2024-04-23T16:17:53.2547294Z ##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1document-processes~1{documentProcessId}~1actions/post/responses/default
2024-04-23T16:17:53.2548989Z ##[error]The connector category is invalid. Available categories are: [AI, Business Management, Business Intelligence, Collaboration, Commerce, Communication, Content and Files, Finance, Data, Human Resources, Internet of Things, IT Operations, Lifestyle and Entertainment, Marketing, Productivity, Sales and CRM, Security, Social Media, Website]. ValuePath: /x-ms-connector-metadata/2/propertyValue
2024-04-23T16:17:53.2550977Z ##[error]The maximum allowed number of categories for the connector is '2'. ValuePath: /x-ms-connector-metadata/2/propertyValue

**Review by vmanoharas — COMMENTED** [review]
Hello @ferenchub

Kindly resolve review comments.

File: certified-connectors/Autenti E-Signature Standard Workflow Connector/README.md (hunk 1 above)
Comment (ferenchub) [reply certified-connectors/Autenti E-Signature Standard Workflow Connector/README.md:2]: Done at commit f9e7ba11477371ff282abad2efb1cd21a77e4735

File: certified-connectors/Autenti E-Signature Standard Workflow Connector/apiDefinition.swagger.json (hunk 2 above)
Comment (ferenchub) [reply certified-connectors/Autenti E-Signature Standard Workflow Connector/apiDefinition.swagger.json:1]: Done at commit 897512aad8dcc50e54f3e6db61b12e94272fa1ee

File: certified-connectors/Autenti E-Signature Standard Workflow Connector/README.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+1. **Trigger**: Initiate the process when the design team uploads final artwork files to a shared drive.
+2. **Action**: Create a new document process for each artwork file, adding them to the document process.
+3. **Action**: Add clients as participants with "Viewer" or "Approver" roles.
+4. **Action**: Begin the approval process, allowing clients to view and approve directly within Autenti.
+5. **Trigger**: Monitor approval status through "Document Change" triggers.
+6. **Action**: Once approval is received, automate the next steps in the project workflow, such as sending the artwork to print or publishing it online.
+
+These examples showcase the versatility of the Autenti E-Signature Connector in Power Automate, enabling organizations to automate complex document workflows, reduce manual tasks, and accelerate decision-making processes.
+
+## Limitations and Known Issues
```
Comment (vmanoharas) [inline certified-connectors/Autenti E-Signature Standard Workflow Connector/README.md]: This heading is not as per the template https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md. Kindly correct this.  I would request you to please check all the headings.

**Review by vmanoharas — COMMENTED** [review]
Hello @ferenchub

Could you please resolve one review comment?

File: certified-connectors/Autenti E-Signature Standard Workflow Connector/README.md (hunk 3 above)
Comment (ferenchub) [reply certified-connectors/Autenti E-Signature Standard Workflow Connector/README.md]: Checked 6de707a3bd6e53bf2c2bd55db7cb40276276a7cd

**Review by vmanoharas — APPROVED** [review]
Hello @ferenchub 
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to Connector certification portal](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in [Connector certification portal](https://connectorcertification.azurewebsites.net/) . Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in [Connector certification portal](https://connectorcertification.azurewebsites.net/), please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Comment by vmanoharas** [comment]
Hello @ferenchub, 

I learnt that you have issue to submit your connector to [Connector certification portal](https://connectorcertification.azurewebsites.net/). Could you please attach the artifacts zip file here.  So we will proceed the certification process.

**Comment by ferenchub** [comment]
Hello @vmanoharas,

Sure, I can send the zip file with the artifacts here, but how should I securely transfer the client_id and client_secret for OAuth2?
Part of the process is providing connection secrets. Thise secrets cannot be shared to GitHub, and under no circumstances can be revealed to third parties. It is also highly not desirable to provide it to Microsoft in a way that enables human ooeratirs to acces them. Please describe the procedure of securely sharing credentials

**Comment by vmanoharas** [comment]
> Hello @vmanoharas,
> 
> Sure, I can send the zip file with the artifacts here, but how should I securely transfer the client_id and client_secret for OAuth2? Part of the process is providing connection secrets. Thise secrets cannot be shared to GitHub, and under no circumstances can be revealed to third parties. It is also highly not desirable to provide it to Microsoft in a way that enables human ooeratirs to acces them. Please describe the procedure of securely sharing credentials

Hello @ferenchub, 

I will give you test account in [Connector certification portal](https://connectorcertification.azurewebsites.net/) to share the required information.

**Comment by ferenchub** [comment]
> > Hello @vmanoharas,
> > Sure, I can send the zip file with the artifacts here, but how should I securely transfer the client_id and client_secret for OAuth2? Part of the process is providing connection secrets. Thise secrets cannot be shared to GitHub, and under no circumstances can be revealed to third parties. It is also highly not desirable to provide it to Microsoft in a way that enables human ooeratirs to acces them. Please describe the procedure of securely sharing credentials
> 
> Hello @ferenchub,
> 
> I will give you test account in [Connector certification portal](https://connectorcertification.azurewebsites.net/) to share the required information.

@vmanoharas 
Okay, in that case, I'll prepare a .zip file. I understand that the archive should contain a file with all the information that was provided on the certification portal? Except for the secrets, of course.

**Comment by vmanoharas** [comment]
> > > Hello @vmanoharas,
> > > Sure, I can send the zip file with the artifacts here, but how should I securely transfer the client_id and client_secret for OAuth2? Part of the process is providing connection secrets. Thise secrets cannot be shared to GitHub, and under no circumstances can be revealed to third parties. It is also highly not desirable to provide it to Microsoft in a way that enables human ooeratirs to acces them. Please describe the procedure of securely sharing credentials
> > 
> > 
> > Hello @ferenchub,
> > I will give you test account in [Connector certification portal](https://connectorcertification.azurewebsites.net/) to share the required information.
> 
> @vmanoharas Okay, in that case, I'll prepare a .zip file. I understand that the archive should contain a file with all the information that was provided on the certification portal? Except for the secrets, of course.

Yes @ferenchub, that's correct, your artifacts should contain intro.md file and icon.png file with 230x230 dimension.

**Comment by ferenchub** [comment]
> > > > Hello @vmanoharas,
> > > > Sure, I can send the zip file with the artifacts here, but how should I securely transfer the client_id and client_secret for OAuth2? Part of the process is providing connection secrets. Thise secrets cannot be shared to GitHub, and under no circumstances can be revealed to third parties. It is also highly not desirable to provide it to Microsoft in a way that enables human ooeratirs to acces them. Please describe the procedure of securely sharing credentials
> > > 
> > > 
> > > Hello @ferenchub,
> > > I will give you test account in [Connector certification portal](https://connectorcertification.azurewebsites.net/) to share the required information.
> > 
> > 
> > @vmanoharas Okay, in that case, I'll prepare a .zip file. I understand that the archive should contain a file with all the information that was provided on the certification portal? Except for the secrets, of course.
> 
> Yes @ferenchub, that's correct, your artifacts should contain intro.md file and icon.png file with 230x230 dimension.


Hello @vmanoharas,

I am sending the archive in the [attachment](https://github.com/microsoft/PowerPlatformConnectors/files/15274131/Autenti.E-Signature.Workflow.zip). Let me know if everything is there and if we can proceed.

**Comment by vmanoharas** [comment]
> > > > > Hello @vmanoharas,
> > > > > Sure, I can send the zip file with the artifacts here, but how should I securely transfer the client_id and client_secret for OAuth2? Part of the process is providing connection secrets. Thise secrets cannot be shared to GitHub, and under no circumstances can be revealed to third parties. It is also highly not desirable to provide it to Microsoft in a way that enables human ooeratirs to acces them. Please describe the procedure of securely sharing credentials
> > > > 
> > > > 
> > > > Hello @ferenchub,
> > > > I will give you test account in [Connector certification portal](https://connectorcertification.azurewebsites.net/) to share the required information.
> > > 
> > > 
> > > @vmanoharas Okay, in that case, I'll prepare a .zip file. I understand that the archive should contain a file with all the information that was provided on the certification portal? Except for the secrets, of course.
> > 
> > 
> > Yes @ferenchub, that's correct, your artifacts should contain intro.md file and icon.png file with 230x230 dimension.
> 
> Hello @vmanoharas,
> 
> I am sending the archive in the [attachment](https://github.com/microsoft/PowerPlatformConnectors/files/15274131/Autenti.E-Signature.Workflow.zip). Let me know if everything is there and if we can proceed.

Hello @ferenchub, thank you very much for submitting the artifacts, can you please modify icon.png into 230x230 dimension and resubmit?

**Comment by ferenchub** [comment]
> > > > > > Hello @vmanoharas,
> > > > > > Sure, I can send the zip file with the artifacts here, but how should I securely transfer the client_id and client_secret for OAuth2? Part of the process is providing connection secrets. Thise secrets cannot be shared to GitHub, and under no circumstances can be revealed to third parties. It is also highly not desirable to provide it to Microsoft in a way that enables human ooeratirs to acces them. Please describe the procedure of securely sharing credentials
> > > > > 
> > > > > 
> > > > > Hello @ferenchub,
> > > > > I will give you test account in [Connector certification portal](https://connectorcertification.azurewebsites.net/) to share the required information.
> > > > 
> > > > 
> > > > @vmanoharas Okay, in that case, I'll prepare a .zip file. I understand that the archive should contain a file with all the information that was provided on the certification portal? Except for the secrets, of course.
> > > 
> > > 
> > > Yes @ferenchub, that's correct, your artifacts should contain intro.md file and icon.png file with 230x230 dimension.
> > 
> > 
> > Hello @vmanoharas,
> > I am sending the archive in the [attachment](https://github.com/microsoft/PowerPlatformConnectors/files/15274131/Autenti.E-Signature.Workflow.zip). Let me know if everything is there and if we can proceed.
> 
> Hello @ferenchub, thank you very much for submitting the artifacts, can you please modify icon.png into 230x230 dimension and resubmit?

Hello @vmanoharas, no problem. Done.
[Autenti E-Signature Workflow v2.zip](https://github.com/microsoft/PowerPlatformConnectors/files/15292372/Autenti.E-Signature.Workflow.v2.zip)

**Comment by ferenchub** [comment]
Hi @vmanoharas , do you have any update on the status of the request?

**Comment by vmanoharas** [comment]
> Hi @vmanoharas , do you have any update on the status of the request?

Hello @ferenchub, 

Thank you for getting back to me, our engineers are working on the submission verification, I will come back to you with 3 to 4 working days with update.

**Comment by vmanoharas** [comment]
Hello @ferenchub, 

I have sent you an email, kindly check and update.

**Comment by ferenchub** [comment]
Hi @vmanoharas , I have checked all my inboxes and haven’t received anything.

**Comment by vmanoharas** [comment]
> Hi @vmanoharas , I have checked all my inboxes and haven’t received anything.

Hi @ferenchub, 

Please check the email given during register of your connector.

**Comment by vmanoharas** [comment]
> > Hi @vmanoharas , I have checked all my inboxes and haven’t received anything.
> 
> Hi @ferenchub,
> 
> Please check the email given during register of your connector.

Hello @ferenchub, 

Can you please look into your email added during your registration and take action, the URL sent will be expired tomorrow 5/23

**Comment by ferenchub** [comment]
Hi, @vmanoharas 
My team and I have been looking for this email for another day now. Could you please tell me the first part of the email address to which you sent the message?

**Comment by vmanoharas** [comment]
Hi @ferenchub, 

This is the first part of the email krzysztof.trojan and domain is your org name.  However, I have added support email from your swagger file.  I hope support team received my email. Please check.

**Comment by ferenchub** [comment]
Hi @vmanoharas,
We are still experiencing the same issue. I am including @DawidCygan in this conversation.
Zrzut ekranu 2024-05-23 o 12 03 20

**Comment by vmanoharas** [comment]
> Hi @vmanoharas, We are still experiencing the same issue. I am including @DawidCygan in this conversation. Zrzut ekranu 2024-05-23 o 12 03 20

Hi @ferenchub, 

Please don't upload new connector, I am requesting to access existing connector where you can find the information regarding how to upload your secrets.  Kindly check my email and reply. We can have a teams call.

**Comment by vmanoharas** [comment]
Hello @ferenchub, 

Kindly ignore the instructions provided above, follow the instructions provided below. 

Congratulations your PR is approved !! We are proud to announce the Brand-New Certification experience for you to certify Power Platform Copilot Connectors & Plugins. Read the blog here- [Announcing Partner Center to certify and publish Power Platform Copilot connectors and plugins](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpowerautomate.microsoft.com%2Fen-us%2Fblog%2Fannouncing-partner-center-to-certify-and-publish-power-platform-copilot-connectors-and-plugins%2F&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390444301%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=5MYg0GlCsic2JqzwJU0h7mIOc7uHOeL7ZWSbgq%2FFkWk%3D&reserved=0)

As next steps: 

1.	Learn about the new certification experience- [Get your Power Platform connector and plugin certified - Overview](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390456966%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=uWDvs%2BmTbozw1oPc%2FNDNWJ%2FHgnUSfFlRCB1T8UHBh3A%3D&reserved=0)
2.	Package your connector and plugin files- [Prepare Power Platform connector and plugin files for certification](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-submission&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390465418%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=pDuv4bnya1eP3Rrs3p4ArNOXw%2FQipWU87q5oZ0VSIe0%3D&reserved=0)
3.	Initiate a certification request in Partner Center- [Verified publisher certification process](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-for-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390471799%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=zbm2px5VZ2ABLBFcORH6tu1fsC9KWdXkP1XJFyluMzw%3D&reserved=0)
4.	Ensure your connector & plugin files comply to the Marketplace policies here- [5000 Power Platform Connector Policies for Marketplace](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Flegal%2Fmarketplace%2Fcertification-policies%235000-power-platform-connectors&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390477736%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=h0pI5J5buo82xxvnBVmrAN8K%2F%2BK8xZlUdcSwzWoqWq0%3D&reserved=0)
5.	Test your connector post certification to provide go-live for deployment- [Testing Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-testing&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390483743%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=Z8pSnhCArFRzEr373o29qM5h5VbV1nlN2gVbv%2Blk1aE%3D&reserved=0)
6.	If you wish to Update your connector or plugin in future- [Updating Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-updates&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390489681%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=CvxTWJFE3bTFeJ3nDJoCSpzvimhVV9go4cBG%2FQ4gMvo%3D&reserved=0)

For any queries or concerns please get in touch with us on [certificationteam@microsoft.com](mailto:certificationteam@microsoft.com)

Many thanks,
Power Platform 
Copilot connector & plugin certification team

**Pull request merged**